joomla/joomla-cms
# [4.0] Get categories from component

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/26157#issuecomment-529189354.

### Summary of Changes

Get categories from component instead of using deprecated `JCategories::getInstance()`.

### Testing Instructions

Create a custom field for articles.
Assign to `Uncategorised` category.
Create an article. Select `Uncategorised` category.
Check that custom field appears in form.

### Expected result AFTER applying this Pull Request

Works like before.

### Documentation Changes Required

No.
